src/bp/core/telemetry/stats/hooks-stats.ts

```diff
@@ -37,7 +37,7 @@ export const getHooksLifecycle = async (
   ghostService: GhostService,
   hashBotId: boolean
 ): Promise<HookPayload> => {
-  const botIds = await botService.getBotsIds()
```

Comment:
Same here!

Reply:
It's still used elsewhere in that file

Reply:
Yes, but `getHooksLifecycle` doesn't require it anymore!

Reply:
Ah right haha. I just checked if the injected service was used, but didn't follow the whole chain. my bad